### PR TITLE
adding /nrdstor/protected

### PR DESCRIPTION
**Pull request by biozit**
adding /nrdstor/protected

### CODE REVIEW

File: virtual-organizations/OSG.yaml [hunk 1]
```diff
@@ -181,6 +181,17 @@ DataFederations:
         AllowedCaches:
           - ANY
 
+      - Path: /nrdstor/protected
+        Authorizations:
+          - SciTokens: https://t.nationalresearchplatform.org/nrdstor
```
Comment (matyasselmeci) [inline virtual-organizations/OSG.yaml:186]:
```suggestion
          - SciTokens:
```

File: virtual-organizations/OSG.yaml [hunk 2]
```diff
@@ -181,6 +181,17 @@ DataFederations:
...
           - ANY
 
+      - Path: /nrdstor/protected
+        Authorizations:
+          - SciTokens:
+              Issuer: https://t.nationalresearchplatform.org
+              Base Path: /nrdstor/protected
+              Map Subject: True
+        AllowedOrigins:
+          - SDSC_NRP_OSDF_S3_ORIGIN
```
Comment (matyasselmeci) [inline virtual-organizations/OSG.yaml:191]: Is this the right origin? Perhaps you want NEBRASKA_NRP_HCC_OSDF_ORIGIN ?

Comment (biozit) [reply virtual-organizations/OSG.yaml]: fixed

**Comment by matyasselmeci** [comment]
Is OSG the right VO for this? NRP.yaml might be more appropriate...

**Comment by biozit** [comment]
> Is OSG the right VO for this? NRP.yaml might be more appropriate...

@djw8605 what you think?

**Review by djw8605 — APPROVED** [review]
Quick question

File: virtual-organizations/OSG.yaml [hunk 3]
```diff
@@ -181,6 +181,17 @@ DataFederations:
         AllowedCaches:
           - ANY
 
+      - Path: /nrdstor/protected
+        Authorizations:
+          - SciTokens:
+              Issuer: https://t.nationalresearchplatform.org
```
Comment (djw8605) [inline virtual-organizations/OSG.yaml:187]: What issuer is this?

**Review by djw8605 — DISMISSED** [review]

**Comment by djw8605** [comment]
This is a NSF CC* Storage OSDF origin.  I think OSG is the appropriate VO for managing that 20% share of the resource.

File: virtual-organizations/OSG.yaml (hunk 3 above)
Comment (brianhlin) [reply virtual-organizations/OSG.yaml:187]: @biozit can you drop a comment here explaining this issuer for posterity?

**Pull request merged**